torch/jit/_script.py

```diff
@@ -418,7 +418,7 @@ def _reconstruct(self, cpp_module):
         def graph(self):
             r"""
             Returns a string representation of the internal graph for the
-            ``forward`` method. See `interpreting-graphs`_ for details.
+            ``forward`` method. See `interpreting-graphs` for details.
```

Comment:
This looks incorrect, and results in broken links (see https://6214508-65600975-gh.circle-artifacts.com/0/docs/generated/torch.jit.ScriptModule.html):

![image](https://user-images.githubusercontent.com/98330/87251619-36e9e900-c46d-11ea-92aa-e4e16bd927d4.png)

you may want `:ref:`?

Reply:
and same for the reference just below

Reply:
fixed